src/Sylius/Bundle/CoreBundle/Resources/config/state-machine.yml

```diff
@@ -180,3 +180,26 @@ winzou_state_machine:
                     to:   'shipped'
                     do:   [@sylius.email_manager.shipment, 'sendConfirmationEmail']
                     args: ['object']
+
+    sylius_review:
```

Comment:
I see that we need to figure out how to configure state machine for "generated" resources. Probably doable through prepending winzou_state_machine configuration - Note for later.